src/App.svelte

```diff
@@ -3,6 +3,10 @@
   import { onMount } from "svelte";
   import authoritiesUrl from "../assets/authorities.geojson?url";
   import type { Schema } from "./types";
+  import bristolUrl from "../assets/bristol.bin?url";
+  import * as Comlink from "comlink";
+  import workerWrapper from "./worker?worker";
```

Comment:
The `?worker` syntax is Vite build magic... which only works in some browsers when doing `npm run dev`. See comments below

Reply:
This is magic to me ✨